source/read/specify-a-query.txt

```diff
@@ -0,0 +1,274 @@
+.. _ruby-specify-query:
+
+===============
+Specify a Query
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: expressions, operations, read, filter, code example
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query by using the {+driver-short+}.
+
+You can refine the set of documents that a query returns by creating a
+**query filter**. A query filter is an expression that specifies the search
+criteria that MongoDB uses to match documents in a read or write operation.
+In a query filter, you can prompt the driver to search for documents that have
+an exact match to your query, or you can compose query filters to express more
+complex matching criteria.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide run operations on the ``fruits`` collection,
+which contains documents representing fruits. The following
+code example shows how to create a database and collection, then
+insert the sample documents into your collection:
```

Comment:
```suggestion
which contains documents representing fruits. The following
code example shows how to create a database and collection, and then
insert the sample documents into your collection:
```
S: Could use the above suggestion or replace the whole thing with just "and". 
Then guidance: https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#t:~:text=two%20days.%20(nonrestrictive)-,then,-When%20then%20is